bpi.module

```diff
@@ -258,6 +272,12 @@ function bpi_form_node_form_alter(&$form, &$form_state, $form_id) {
     $teaser = variable_get('bpi_field_teaser', '');
     $body = variable_get('bpi_field_body', '');
     $materials_field = variable_get('bpi_field_materials', '');
+
+    // @todo Current language will be set to undefined, because this module is
+    // lexigraphically before the locale module. That makes the implementation
+    // less robust, and maybe this should be changed. When $current_language
+    // is set to the correct current language, the code will not work and the
+    // fields will not be populated with data from BPI Syndication.
```

Comment:
It's not that simple. The problem is, that if the BPI module is run after locale the language will be set to the current language. But if that happens, the fields are not populated with the data from BPI Syndication, because the fields are set to undefined and not the node's language. That's why I made this todo, cause it's not obvious for me what is supposed to be done.